docs/cita/node/cons-node.md

```diff
@@ -63,72 +63,72 @@ $ cita-cli scm NodeManager listNode --url http://127.0.0.1:1337
 
 * 发送交易
 
-  ```bash
-  $ cita-cli scm NodeManager approveNode \
-      --address 0x59a316df602568957f47973332f1f85ae1e2e75e \
-      --admin-private 0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6 \
-      --url http://127.0.0.1:1337
-  ```
+```bash
```

Comment:
Does this have the format problem?

Reply:
<img width="656" alt="image" src="https://user-images.githubusercontent.com/42529063/63676142-e9489b80-c81c-11e9-8f88-4a94ed7ef5f6.png">